.github/workflows/run-e2e-tests.yaml

```diff
@@ -17,6 +17,10 @@ jobs:
           browser: chrome
           wait-on: 'http://localhost:3000'
           config: baseUrl=http://localhost:3000
+          command: npm run cypress:percy
+        env:
+          # pass the Percy Token
+          PERCY_TOKEN: ${{ secrets.PERCY_TOKEN }}
```

Comment:
So this is something that can only be run on CICD? Will it gracefully fail if we run cypress locally?

Reply:
Only the GitHub action uses percy. Locally you run cypress in your own browser via the `cypres:run` command.

Reply:
There are test lines which include calls to Percy - just making sure these won't fail.

Reply:
Dev told me that these percy commands are just skipped and do not throw if you do not have the percy token. 👍🏼